web/client/utils/mapinfo/__tests__/wms-test.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
```

Comment:
this is a minor thing, but for new files the copyright has to be aligned to current year
```suggestion
 * Copyright 2023, GeoSolutions Sas.
```